### DISCUSSION TITLE
Classifiers on PyPI to help projects declare they are compatible with WebAssembly

### DISCUSSION THREAD

**Discussion by brettcannon** [pyodide/pyodide#3480] [Show and tell]
There are now three new [classifiers on PyPI](https://pypi.org/classifiers/) related to WebAssembly:
1. Environment :: WebAssembly
2. Environment :: WebAssembly :: Emscripten
3. Environment :: WebAssembly :: WASI

The idea is that any project that works with WebAssembly out-of-the-box can add this classifier to make it easier to find them (i.e. pure Python wheel projects that have verified they work with a WebAssembly runtime).

If a project works with both Emscripten and WASI, the more generic `Environment :: WebAssembly` should be used. Otherwise, the specific WebAssembly target that the project happens to be compatible with should be specified.

**Comment by rth** [comment]
Thanks a lot for doing this @brettcannon ! We will add it to the docs.
